packages/gcp/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.38.0"
+  changes:
+    - description: Add `policy_violation_info`, `metadata` and `related` fields to audit logs. Update GCP audit log dashboard to use correct `email` field.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/10886
```

```suggestion
    - description: Add `policy_violation_info`, `metadata` and `related` fields to audit logs.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/10886
         - description: Update GCP audit log dashboard to use correct `email` field.
      type: bugfix
      link: https://github.com/elastic/integrations/pull/10886
```

Since the two things being changes are seemingly unrelated, it's nice to break them up.